edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/DeviceScopeApiClient.cs

```diff
@@ -20,8 +21,12 @@ public class DeviceScopeApiClient : IDeviceScopeApiClient
 
         const string InScopeTargetIdentityUriFormat = "/devices/{0}/modules/{1}/deviceAndModuleInDeviceScope?targetDeviceId={2}&targetModuleId={3}&api-version={4}";
 
+        const string GetDevicesAndModulesInTargetScopeUriFormat = "/devices/{0}/modules/{1}/devicesAndModulesInTargetDeviceScope?api-version={2}";
+
         const string ApiVersion = "2018-08-30-preview";
 
+        const string NestedApiVersion = "2020-06-30-preview";
```

Comment:
I expected the new api version should be backward compatible for existing API calls, and only new api version should be used.